x/lockup/keeper/behaviour.go

```diff
@@ -0,0 +1,309 @@
+package keeper
+
```

Comment:
lets add a todo for moving this to either an internaal folder or simulation folder, in the future. (Fine to be here for launch)